torchaudio/transforms.py

```diff
@@ -869,3 +871,34 @@ def forward(self, waveform: Tensor) -> Tensor:
             waveform = F.gain(waveform, 10 * math.log10(self.gain))
 
         return torch.clamp(waveform, -1, 1)
+
+
+class SlidingWindowCmn(torch.nn.Module):
+    r"""
+    Apply sliding-window cepstral mean (and optionally variance) normalization per utterance.
+
+    Args:
+        cmvn_window (int, optional): Window in frames for running average CMN computation (int, default = 600)
+        min_cmn_window (int, optional):  Minimum CMN window used at start of decoding (adds latency only at start). Only applicable if center == false, ignored if center==true (int, default = 100)
+        center (bool, optional): If true, use a window centered on the current frame (to the extent possible, modulo end effects). If false, window is to the left. (bool, default = false)
+        norm_vars (bool, optional): If true, normalize variance to one. (bool, default = false)
+    """
+
+    def __init__(self, cmvn_window: int = 600, min_cmn_window: int = 100, center: bool = False, norm_vars: bool = False):
+        super(SlidingWindowCmn, self).__init__()
```

Comment:
We have dropped support for Python 2. So this can be simplified to `super().__init__()`

Reply:
ths